src/planner.rs

```diff
@@ -18,12 +15,9 @@ impl RunArg {
     pub fn fetch<'a, U, F>(&'a self, f: F) -> U
         where F: FnOnce(&'a World) -> U
     {
-        let pulse = self.pulse.borrow_mut().take()
-                        .expect("fetch may only be called once.");
-        let u = f(&self.world);
-        pulse.pulse();
-        u
+        f(&self.world)
```

Comment:
> Do you mean sequentialized?

Yes. Basically, the planner fires up the systems one by one. It waits for a system to finish the `fetch` state before proceeding to the next one.